scripts/buildkite/stack-cabal_config_check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -euo pipefail
+
+# This script checks that the `stack.yaml` and `cabal.project` files have
+# consistent git hashes for the packages they depend on. We use
+# `cardano-repo-tool`'s `update-cabal-project` command which modifies
+# `cabal.project` to be consistent with `stack.yaml`s versions. If the
+# diff is non-empty, we know they're out of sync.
```

Comment:
This description is the same as in the `scripts/buildkite/stack-cabal-sync.sh` script which suggests that one is not needed.

Reply:
we're upstreaming that to iohk-nix currently. Will push as soon as that merges.